packages/data-driver-postgres/src/query/index.ts

```diff
@@ -1,7 +1,25 @@
-import type { SqlStatement } from '@directus/data-sql';
+import type { AbstractSqlQuery } from '@directus/data-sql';
 import { from } from './from.js';
 import { select } from './select.js';
+import type { ParameterizedSQLStatement } from '@directus/data-sql';
+import { limit } from './limit.js';
+import { offset } from './offset.js';
 
-export function constructSql(query: SqlStatement): string {
-	return [select(query), from(query)].join(' ') + ';';
+/**
+ * All of the sub functions are called for any query.
+ * Within those functions checks are being made, if the part can or should be included within the statement.
```

Comment:
While I think we can leave that description of the inner functionality of `constructSqlQuery` here, I'd at least add a short description about the general purpose of `constructSqlQuery` on top of that.

Reply:
I removed the inner functionality description for now and replaced it with the general purpose. In another branch I add a bit better description for the inner functionality